Spanner/src/Session/CacheSessionPool.php

```diff
@@ -718,7 +722,11 @@ private function isSessionValid(array $session)
     {
         $halfHourBeforeExpiration = $session['expiration'] - (SessionPoolInterface::SESSION_EXPIRATION_SECONDS / 2);
```

Comment:
rant: this is strange to write it like this... I think a better way would be
```suggestion
        $halfHourBeforeExpiration = $session['expiration'] - (SessionPoolInterface::SESSION_EXPIRATION_SECONDS - 1800;
```
I know that doesn't have to do with this PR though.

Reply:
I'll add this change too.